src/Workspaces/Core/Portable/Workspace/Host/TemporaryStorage/ITemporaryStorage.cs

```diff
@@ -29,14 +29,3 @@ public interface ITemporaryStreamStorage : IDisposable
     void WriteStream(Stream stream, CancellationToken cancellationToken = default);
     Task WriteStreamAsync(Stream stream, CancellationToken cancellationToken = default);
 }
-
-/// <summary>
```

Comment:
Have the interfaces and their implementations above been obsoleted long enough for removal?

Reply:
We have to make then `Obsolete(error: true)` first.  i can do that in a followup.

Reply:
opened: https://github.com/dotnet/roslyn/pull/73226